public/locales/en/panel-time.json

```diff
@@ -0,0 +1,68 @@
+{
+  "select-time-title": "Select Time",
+  "error": {
+    "outside-range": "Can't interact with dates outside the range April 20, 271821 BC to Sep 13, 275760 AD."
+  },
+  "simulation-speed-title": "Simulation Speed",
+  "realtime": "Realtime",
+  "now": "Now",
+  "time-parts": {
+    "Milliseconds_one": "Milliseconds",
+    "Milliseconds_other": "Milliseconds",
+    "Seconds_one": "Second",
+    "Seconds_other": "Seconds",
+    "Minutes_one": "Minute",
+    "Minutes_other": "Minutes",
+    "Hours_one": "Hour",
+    "Hours_other": "Hours",
+    "Days_one": "Day",
+    "Days_other": "Days",
+    "Weeks_one": "Week",
+    "Weeks_other": "Weeks",
+    "Months_one": "Month",
+    "Months_other": "Months",
+    "Years_one": "Year",
+    "Years_other": "Years"
+  },
+  "simulation-increment": {
+    "select-unit-label": "Display Unit"
+  },
+  "quick-adjust": {
+    "title": "Quick Adjust"
+  },
+  "delta-time-step-control": {
+    "aria-label": {
+      "previous-step": "Set previous delta time step",
+      "next-step": "Set next delta time step",
+      "toggle-pause": {
+        "play": "Play time",
+        "pause": "Pause time"
+      }
+    }
+  },
+  "time-input": {
+    "error": {
+      "invalid-time": {
+        "title": "Invalid time",
+        "message": "Error trying to set time to an invalid date"
+      },
+      "invalid-date": "New date is outside range (April 20, 271821 BC to Sep 13, 275760 AD)",
+      "invalid-year": "Year '{{value}}' is outside allowed year range (April 20, 271821 BC to Sep 13, 275760 AD)"
+    },
+    "lock-time": {
+      "tooltip": "Lock time updates to prevent automatic changes. Use 'Interpolate' or 'Set' to manually apply the selected date and time.",
```

Comment:
Seems like it would potentially be a place where we want to refer to the `button-labels` keys instead of repeating the words? To prevent the risk of a mismatch?